### PR TITLE
Fix #546 wrong provider credentials

### PR DESCRIPTION
**Pull request by s0undt3ch**

### CODE REVIEW

**Comment by s0undt3ch** [comment]
@techhat Ready for review again.

**Comment by s0undt3ch** [comment]
Ready once more.

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # pip requirements file for Salt Cloud
-salt
+salt>=0.16.0
```
Comment (techhat) [inline requirements.txt]: Which issue is it that 0.16.0 fixes?

Comment (s0undt3ch) [inline requirements.txt]: Json traceback.

## 

Pedro Algarvio @ phone

----- Reply message -----
De: "Joseph Hall" notifications@github.com
Para: "saltstack/salt-cloud" salt-cloud@noreply.github.com
Cc: "Pedro Algarvio" pedro@algarvio.me
Assunto: [salt-cloud] Fix #546 wrong provider credentials (#635)
Data: ter, Jun 18, 2013 23:02
In requirements.txt: 

> @@ -1,4 +1,4 @@ 
>  # pip requirements file for Salt Cloud 
> -salt 
> +salt>=0.16.0 

Which issue is it that 0.16.0 fixes? 

&mdash;

Comment (s0undt3ch) [inline requirements.txt]: We can remove that if you prefer.

## 

Pedro Algarvio @ phone

----- Reply message -----
De: "Joseph Hall" notifications@github.com
Para: "saltstack/salt-cloud" salt-cloud@noreply.github.com
Cc: "Pedro Algarvio" pedro@algarvio.me
Assunto: [salt-cloud] Fix #546 wrong provider credentials (#635)
Data: ter, Jun 18, 2013 23:02
In requirements.txt: 

> @@ -1,4 +1,4 @@ 
>  # pip requirements file for Salt Cloud 
> -salt 
> +salt>=0.16.0 

Which issue is it that 0.16.0 fixes? 

&mdash;

Comment (techhat) [inline requirements.txt]: I would like to remove it.

**Comment by s0undt3ch** [comment]
Removed the 0.16.0 hard dep

**Pull request merged**